### PR TITLE
[thorfinn] model-slices sweep {64, 128, 192} on baseline

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

`--model-slices` controls the number of slice tokens that AB-UPT uses to attend over the surface point cloud. The current bengio baseline uses the default (likely 96). Wave 1 mid-training data (kohaku 128 slices) showed a small improvement vs default at mid-training; gilbert (5L) used 192 (PR #188 stack). The slice count interacts with the wsy/wsz boundary-layer attention quality — more slices = finer surface resolution.

This is a cheap capacity sweep on the existing baseline recipe (no code change needed, just a CLI flag). Hypothesis: increasing slices to 128 or 192 reduces wall-shear errors (especially wsy/wsz on complex geometries) at modest training-cost increase.

## Instructions

Run a 3-trial sweep using `--wandb-group bengio-wave4-slices-sweep`. Confirm the default value with `python train.py --help | grep slices` and set the trials to span around it.

**Trial A — `--model-slices 128` (kohaku Wave 1 value)**:
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 4 --model-hidden-dim 256 --model-heads 4 \
  --no-use-ema --lr 3e-4 --lr-cosine-t-max 30 --no-compile-model \
  --model-slices 128 \
  --kill-thresholds "35000:val_primary/abupt_axis_mean_rel_l2_pct<20" \
  --wandb-group bengio-wave4-slices-sweep
```

**Trial B — `--model-slices 192`**: same with `--model-slices 192`.

**Trial C — `--model-slices 64` (lower-bound check)**: same with `--model-slices 64`. Only if A and B both finish.

**Gates**: ep10<10.0, ep20<8.5, ep30<7.5.

**OOM contingency**: If 192 slices OOMs at 4L/256d/4H, drop `--batch-size` (find current default and halve). Report the working batch size in PR comments.

Report ep30 val_abupt + per-axis decomposition for each trial. Key signal: monotone-or-not behavior in slices→accuracy. If 128 is best (non-monotone), there's a sweet spot to find.

## Baseline

Current best (from `target/BASELINE.md`):
- `val_primary/abupt_axis_mean_rel_l2_pct`: **7.2091**
- `val_primary/wall_shear_rel_l2_pct`: 8.160 (target 7.29)
- `val_primary/wall_shear_y_rel_l2_pct`: 9.100 (target 3.65 — binding)
- `val_primary/wall_shear_z_rel_l2_pct`: 10.869 (target 3.63 — binding)
- `val_primary/surface_pressure_rel_l2_pct`: 4.802
- `val_primary/volume_pressure_rel_l2_pct`: **4.166**

Reproduce baseline: 4L/256d + ContinuousSincosEmbed + no-EMA + cosine T_max=30 + lr=3e-4 + default model_slices.

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR — **URGENT: Launch required now**

This PR has been open for >24 hours with no student comment. The model-slices sweep is a no-code-change Wave 4 experiment — just CLI flags. Please launch immediately.

First, check the default slices value:
```bash
cd target/ && python train.py --help | grep slices
```

**Trial A — `--model-slices 128`:**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 4 --model-hidden-dim 256 --model-heads 4 \
  --no-use-ema --lr 3e-4 --lr-cosine-t-max 30 --no-compile-model \
  --model-slices 128 \
  --kill-thresholds "35000:val_primary/abupt_axis_mean_rel_l2_pct<20" \
  --wandb-group bengio-wave4-slices-sweep
```

**Trial B — `--model-slices 192`:** Queue after Trial A clears ep5. If Trial B OOMs, halve batch size and report.

Post the W&B run ID and confirm the default slices value in a PR comment ASAP.

Gates: ep10<10.0, ep20<8.5, ep30<7.5.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #260 because ADVISOR: Closing after two URGENT notices with no response in >48 hours. Experiment was model-slices sweep {128, 192} vs default. Hypothesis remains valid and will be reassigned..
